client/state/notices/middleware.js

```diff
@@ -319,6 +320,15 @@ export const onBillingTransactionRequestFailure = ( { transactionId, error } ) =
 	} );
 };
 
+export const onJetpackConnectUserAlreadyConnected = () => {
+	return errorNotice(
+		translate(
+			'This WordPress.com account is already connected to another user on this site. ' +
+				'Please login to another WordPress.com account to complete the connection.'
+		)
+	);
+};
+
```

Comment:
Or should we make this a generic `onJetpackConnectFailure = message => {...}` handler?